tests/utils/installer.py

```diff
@@ -103,7 +103,7 @@ def check_if_installed():
 
 def uninstall_iotrace():
     TestRun.LOGGER.info("Uninstalling previous iotrace")
-    TestRun.executor.run_expect_success(
+    TestRun.executor.run(
```

Comment:
This step used to fail entire execution when installting iotrace on a fresh system - cd failed.